### ISSUE TITLE
Login does not allow to save credentials in browser

### ISSUE THREAD

**Issue by pschichtel** [ludumdare/ludumdare#266]
I'd personally like to store my credentials in Firefox' credential store, but the way ldjam's login is done, does not trigger Firefox' "Save credentials" popup.

**Comment by WizzardMaker** [comment]
It seems that its just a firefox issue, I use Chrome and it toggles the popup to save my password and username. It pastes the right values while logging in too

**Comment by povrazor** [comment]
Yeah, I've only really had a chance to test this in Chrome. Thanks for the report.

**Comment by karai17** [comment]
I'd like to add that the sign up page and the login page don't interact correctly with lastpass. When signing up, I had to manually generate a password since clicking the little lastpass icon in the text box wasn't working. When logging in, I need to manually select which login to use instead of it auto populating.

**Comment by povrazor** [comment]
Noted. Thanks.
